numerical_methods/false_position.cpp

```diff
@@ -19,56 +23,87 @@
 #include <iostream>
 #include <limits>
 
-#define EPSILON \
-    1e-6  // std::numeric_limits<double>::epsilon()  ///< system accuracy limit
-#define MAX_ITERATIONS 50000  ///< Maximum number of iterations to check
+int RANGE = 100000;  ///< Range in which we have to find the root. (-Range,Range)
+double INTERVAL_GAP = 0.5; /// interval gap. lesser the gap more the accuracy
```

Comment:
Avoid using global variables, they may cause overflows and other issues.
Pass them as local function parameters, or as class private members.

Reply:
Removed Global Variables. Added local variables instead. Please have a look @Panquesito7.